src/components/templates/PosterPage/PosterPage.tsx

```diff
@@ -77,7 +77,9 @@ export const PosterPage: React.FC<PosterPageProps> = ({ venue }) => {
         <div />
 
         <div className="PosterPage__header--middle-cell">
-          <p className="PosterPage__title">{title}</p>
+          <p className="PosterPage__title">
+            {venue.name}: {title}
+          </p>
```

Comment:
@sparkletown/product @mike-lvov Would you be happy with this change being made (to show the `posterVenue` name in full + the poster title) instead of https://github.com/sparkletown/internal-sparkle-issues/issues/755 (which isn't implementable in a generalisable way as we don't (to my knowledge) have a concept of 'poster number' outside of the implied form of it through `posterVenue.name`)

The changes in this file represent how it's displayed in the `PosterPage` venue itself.

And if so, would you be happy for that to be a change made/merged into our core codebase? Or would we do this as an `env/ohbm` specific hack?

Reply:
@sparkletown/product @mike-lvov pretty please?

Reply:
@sofisparkle ?

Reply:
Looks good

Reply:
>  Or would we do this as an env/ohbm specific hack?

OHBM specific hack as venueName is not restricted to be Poster{NNN}